texme.js

```diff
@@ -174,6 +174,9 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
       '    margin: 1em auto;',
       '    box-shadow: 5px 5px 5px #222;',
       '  }',
+      '  img {',
+      '    max-width: 40em;',
+      '  }',
       '}'
```

Comment:
Hmmm after testing it out, I think I should keep the max-width to 48 em for the @media screen but above this block something like
`max-width: 80%;` might be good since the image will scale based on how small your device is.

here's a GIF example: https://gfycat.com/recklessflusteredbass

please let me know what you think

Reply:
Hi @AceroM,

For the `@media screen and (min-width: 40em)` block, `max-width: 48em` does not sound right. The image could then be wider than the content width (width of `main` minus padding) or even go outside the white pane (`main`) itself and overflow into the gray background because the CSS for the white pane (`main`) is set to `max-width: 40em`.

You can test this happening as you shrink the browser gradually. Just when the browser shrinks just enough that the `main` element is say only 640 px wide (i.e., 40 em wide) and the browser is still wider than 48 em, a large image would be `48em` wide which would exceed the white content pane (`main`). Here is an example: https://imgur.com/a/IcpGJZK

Did you mean `max-width: 40em` instead? That suffers from a similar problem too. For the `@media screen and (min-width: 40em)` block, it is possible that the content width (the width of the area where you see the text) is smaller than 40em because the content width is width of `main` minus padding. So if the `img` `max-width` is set to `40em` it allows the image width to exceed the content width. Here is an example: https://imgur.com/a/KOlFRLr

I am curious why you want two separate values for the `@media screen` block and outside the `@media screen` block?

I've tested that a single value of `max-width: 100%` outside the `@media screen` block works fine. This value ensures that the image cannot exceed the content width (width of `main` minus padding) in all screen sizes (whether the screen size exceeds 40em or not). It also ensures that the image scales down based on how small the device is. Further the image width never exceeds the content width. It's a simpler solution that works correctly for all browser sizes, therefore I prefer this.

I understand that someone may prefer the image to be slightly narrower than the content width, say `max-width: 80%` or `max-width: 90%`. Such personal preferences can be addressed by custom styles ([styles](https://github.com/susam/texme#style) &raquo; second example).